src/utils/map.ts

```diff
@@ -60,21 +60,25 @@ export module Utils {
     }
 
     /**
-     * Return an array of the values in the Map
+     * The forEach method executes the provided callback once for each key of the map which
+     * actually exist. It is not invoked for keys which have been deleted.
+     * However, it is executed for values which are present but have the value undefined.
      *
-     * @return {V[]} The values in the store
-     */
-    public values() {
-      return this._keyValuePairs.map((keyValuePair) => keyValuePair.value);
-    }
-
-    /**
-     * Return an array of keys in the Map.
+     * Callback is invoked with three arguments:
+     *   - the element value
+     *   - the element key
+     *   - the Map object being traversed
      *
-     * @return {K[]} The keys in the store
+     * @param {Function} callbackFn The callback to be invoked
+     * @param {any} thisArg The `this` context
      */
-    public keys() {
-      return this._keyValuePairs.map((keyValuePair) => keyValuePair.key);
```

Comment:
No, it is not part of the ES6 spec. The implementation of `.keys()` and `.values()` is different.